apps/emqx_bridge_s3tables/test/emqx_bridge_s3tables_SUITE.erl

```diff
@@ -1056,6 +1120,499 @@ t_write_after_delete(Config) ->
 
     ok.
 
+%% Here, in the test cases, we need to use emulator container for S3{,Tables}, hence we
+%% pass the base endpoints directly as hidden parameters.  This test cases checks the
+%% normal, production code path of inferring AWS endpoints from the provided S3Tables ARN.
+t_start_connector_only_with_arn(Config0) ->
+    Config = emqx_bridge_v2_testlib:proplist_update(Config0, connector_config, fun(Old) ->
+        Cfg = maps:without([<<"account_id">>, <<"bucket">>, <<"base_endpoint">>], Old),
+        maps:update_with(
+            <<"s3_client">>,
+            fun(S3Client0) ->
+                S3Client = maps:without([<<"host">>, <<"port">>], S3Client0),
+                emqx_utils_maps:deep_merge(
+                    S3Client,
+                    #{<<"transport_options">> => #{<<"ssl">> => #{<<"enable">> => true}}}
+                )
+            end,
+            Cfg
+        )
+    end),
+    on_exit(fun meck:unload/0),
```

Comment:
Nit: probably not needed,`meck:new/2` is called without `no_link`

Reply:
I place those out of habit :joy_cat: